src/archivematicaCommon/tests/test_config.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+import os
+import StringIO
+import ConfigParser
+
+from django.test import TestCase
+import pytest
+
+from env_configparser import EnvConfigParser
+
+
+class TestConfigReader(TestCase):
+    def setUp(self):
+        """
+        Make sure that we are not mutating the global environment.
+        """
+        self.environ = os.environ.copy()
```

Comment:
You probably want `copy.deepcopy()` since `os.environ` is probably compound.  https://docs.python.org/2/library/copy.html